src/app/repository.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RepositoryService {
+  constructor() {}
```

Comment:
why is it empty? and why service is located in root dir? all GLOBAL services are located under `app/services/` folder

Reply:
woops that's not supposed to be here

Reply:
This service should not be here. I deleted it.